src/class-wc-gateway-btcpay.php

```diff
@@ -735,8 +736,18 @@ public function process_payment($order_id)
                 $this->log('    [Info] Invoice object created successfully...');
             }
 
-            $order_number = $order->get_order_number();
-            $invoice->setOrderId((string)$order_number);
+            $order_url = $order->get_edit_order_url();
+
+            $pos_data = array(
+              'Woocommerce Order ID' => $order_id,
+              'Woocommerce Order Number' => $order_number,
+              'Woocommerce Order URL' => $order_url,
+              'Woocommerce Thanks URL' => $thanks_link,
```

Comment:
Not sure if $thanks_link is interesting here, from looking above it can end up to be the same as $redirect_url. But I would at least also add the $notification_url to see what callback was used? Anyway, no hard feelings here, just a thought. Feel free to ignore :)

Reply:
Removed the thanks link in a2bf31f6c8f6f6245bca0956390b33b7c55bb4a6.
I don't think we need the notification URL here, as it is also part of the debug log.
Trimmed the links down to just the one that gives the user a link back to the WooCommerce backend.